benchmark/v8/get-stats.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common.js');
+const v8 = require('v8');
+
+const bench = common.createBenchmark(main, {
+  method: [
+    'getHeapStatistics',
+    'getHeapSpaceStatistics'
+  ],
+  n: [1e6],
+  flags: ['--ignition --turbo', '']
```

Comment:
If I understand the [guide](https://github.com/nodejs/node/blob/master/doc/guides/writing-and-running-benchmarks.md#basics-of-a-benchmark), this should be a property of the third argument (`options`). And flags does not vary, so empty flag seems to be useless. However, I may miss something.